site/_includes/header.html

```diff
@@ -36,7 +36,8 @@
               <a href="/blog">Blog</a>
             </li>
             <li><a href="https://github.com/bazelbuild/bazel">GitHub</a></li>
```

Comment:
It's important that GitHub appears more prominently than Twitter, Stack Overflow or Mailing list. When I tried to use an icon, it was less noticeable, thus I kept it as a plain word.

Do you think we should aim for using a maximum of icons?